components/common/HapticTab.tsx

```diff
@@ -1,13 +1,13 @@
-import { BottomTabBarButtonProps } from '@react-navigation/bottom-tabs';
-import { PlatformPressable } from '@react-navigation/elements';
-import * as Haptics from 'expo-haptics';
+import { BottomTabBarButtonProps } from "@react-navigation/bottom-tabs";
+import { PlatformPressable } from "@react-navigation/elements";
+import * as Haptics from "expo-haptics";
 
 export function HapticTab(props: BottomTabBarButtonProps) {
   return (
     <PlatformPressable
       {...props}
       onPressIn={(ev) => {
-        if (process.env.EXPO_OS === 'ios') {
+        if (process.env.EXPO_OS === "ios") {
```

Comment:
```suggestion
        if (Platform.OS === 'ios') {
```

https://reactnative.dev/docs/0.77/platform-specific-code

Reply:
That's component generated by expo template project but is it bad to reference to this env variable as we working on Expo project instead of RN Platform API? Or is it just we prefer framework-agnostic way?